### PR TITLE
new sql content

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by caupolicandiaz** [comment]
I would submit that the data types table is admittedly limited, and simply based on the types that were included on the Monday board. I'm happy to expand this to include any other types that are recommended. Unfortunately, this table is complicated by the fact that there are many differences in the supported types among SQL dialects, I chose arbitrarily, to show examples from Postgres. I'm happy to hear any alternative ways of organizing and displaying types.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@caupolicandiaz Finished with my review and left you some comment/suggestions.

**Comment by caupolicandiaz** [comment]
@Dusch4593 I've incorporated all your edits, ready for another review. Thanks

**Review by Dusch4593 — APPROVED** [review]
@caupolicandiaz LGTM for the next review! 👍🏻

File: content/sql/concepts/data-types/data-types.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Numeric types
+- String or character types
+- Temporal types for times and dates
+
+These data types are found across all flavors of SQl; however, some versions of SQL may support several distinct data types within a class while others may only have one (the definitions below are specific to PostgreSQL).
+
+- `integer`: A whole number between -2147483648 and 2147483647. Postgres also includes alternatives `smallint` and `bigint`.
+- `real`: A floating-point type that has variable-precision with a maximum range of 6 decimals.
+- `text`: A range of characters of unlimited length.
+- `date`: A date (without any time value), examples include 2022-06-21 (ISO 8601 format) and 6/21/2022.
```
Comment (SSwiniarski) [inline content/sql/concepts/data-types/data-types.md:29]: I'd add CHAR and VARCHAR data types as they're common across SQL flavors.

File: content/sql/concepts/data-types/data-types.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**Data types** provide categories for values we store in tables. Types are assigned to fields through table creation and are responsible for determining some of the attributes and constraints (such as the amount of memory allocated) of data stored within a given table.
+
+SQL supports a range of data types across widely used classes of data, such as the following:
+
+- Numeric types
+- String or character types
+- Temporal types for times and dates
+
+These data types are found across all flavors of SQl; however, some versions of SQL may support several distinct data types within a class while others may only have one (the definitions below are specific to PostgreSQL).
```
Comment (SSwiniarski) [inline content/sql/concepts/data-types/data-types.md:24]:
```suggestion
These data types are found across all flavors of SQl; however, some versions of SQL may support several distinct data types of a particular class while others may only have one (the definitions below are specific to PostgreSQL).
```

File: content/sql/concepts/data-types/data-types.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Data Types'
+Description: 'Data types provide categories for values we store in tables'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Types'
+  - 'Documentation'
```
Comment (SSwiniarski) [inline content/sql/concepts/data-types/data-types.md:9]:
```suggestion
```

File: content/sql/concepts/commands/terms/round/round.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+The **`ROUND`** command rounds a value to the nearest integer or to a specific number of decimals if an optional value is provided.
+
+## Syntax
+
+```pseudo
+ROUND(value, decimal_places)
+```
+
+## Examples
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/round/round.md:25]:
```suggestion
## Example
```

File: content/sql/concepts/commands/terms/round/round.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'ROUND'
+Description: 'Rounds a value to the nearest integer or to a specific number of decimals if an optional value is provided.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Types'
+  - 'Documentation'
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/round/round.md:9]:
```suggestion
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@caupolicandiaz, I finished a review and left a few comments.

**Comment by caupolicandiaz** [comment]
@SSwiniarski @Dusch4593 Thanks for edits, I've made all the updates in response to your comments. Ready for the next review

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz, Looks good on my end.

**Comment by Dusch4593** [comment]
This looks ready to merge! 🚀 I made some minor edits in b79f37fe685bd64635a713327030a66c30b1819e.

**Pull request merged**